analytics/mapreduce/src/main/java/mil/nga/giat/geowave/analytic/mapreduce/kde/KDEJobRunner.java

```diff
@@ -205,7 +209,9 @@ public int runJob()
 			statsReducer.setMapperClass(IdentityMapper.class);
 			statsReducer.setPartitionerClass(getJob2Partitioner());
 			statsReducer.setReducerClass(getJob2Reducer());
-			statsReducer.setNumReduceTasks(getJob2NumReducers((kdeCommandLineOptions.getMaxLevel() - kdeCommandLineOptions.getMinLevel()) + 1));
+			statsReducer
+					.setNumReduceTasks(getJob2NumReducers((kdeCommandLineOptions.getMaxLevel() - kdeCommandLineOptions
```

Comment:
Same as above, this one is being caused by:
`<setting id="org.eclipse.jdt.core.formatter.lineSplit" value="120"/>`